File: MY PROGRAMS/LB/JAVA/NUMBER/a6q5.java
/*	5. Write a program which accept total marks & obtained marks from user and calculate percentage.

	Input:	1000 745
	Output:	74.5%	*/




import java.util.*;
import java.io.*;
import java.lang.*;

class Assignment6
{
	public float Return(float ftot,float fobt)
	{
		float fper,fmarks;

		fmarks=fobt/ftot;
		fper=fmarks*100;

		return(fper);
	}
}

class a6q5
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		float fvalue1,fvalue2,fret;

		System.out.println("Enter the total and obtained marks: ");
		fvalue1=sobj.nextFloat();
		fvalue2=sobj.nextFloat();

		Assignment6 aobj=new Assignment6();

		fret=aobj.Return(fvalue1,fvalue2);

		System.out.println("Percentage is "+fret+"%");
	}
}

File: MY PROGRAMS/LB/CPP/SELECTION,SEQUENCE AND  ITERATION/a6q5.cpp
/* Write a program which accept total marks & obtained marks from user and calculate percentage.

Input:	1000 745
Output:	74.5%

*/




#include<iostream>

using namespace std;

class Percentage
{
	public:
		float Return(float,float);
};

float Percentage::Return(float ftot,float iobt)
{
	float fper=0.0;

	fper=(iobt*100)/ftot;

	return(fper);
}

int main()
{
	 float fret=0.0,ivalue1=0.0,ivalue2=0.0;

	cout<<"Enter the total and obtained marks of student:";
	cin>>ivalue1>>ivalue2;

	Percentage pobj;

	fret=pobj.Return(ivalue1,ivalue2);

	cout<<"Percentage of a student is "<<fret<<"%";

	return(0);
}
